client/influxdb.go

```diff
@@ -112,13 +99,21 @@ func (c *Client) Write(writes ...Write) (*Results, error) {
 	defer resp.Body.Close()
 
 	var results Results
-	dec := json.NewDecoder(resp.Body)
-	dec.UseNumber()
-	err = dec.Decode(&results)
+	body, _ := ioutil.ReadAll(resp.Body)
+	if len(body) > 0 {
```

Comment:
For this endpoint, if it's successful (meaning we wrote the data), we just respond with a status code 200, there is no body to decode.